src/main/java/world/bentobox/bentobox/util/Util.java

```diff
@@ -656,4 +660,18 @@ public static void resetHealth(Player player) {
         double maxHealth = player.getAttribute(Attribute.GENERIC_MAX_HEALTH).getBaseValue();
         player.setHealth(maxHealth);
     }
+
+    /**
+     * Prettify and try to localize an entity type
+     * @param entityType - entity type
+     * @param string - locale to localize to, if possible
+     * @return string
+     */
+    public static String prettifyText(EntityType entityType, String string) {
+        if (Bukkit.getPluginManager().getPlugin("LangUtils") != null) {
```

Comment:
That might be a bit "overkill", but the Hooks API is designed to support that kind of checks, and it makes the code clearer by handling the "presence" of the plugin in a more abstract way. Mind giving that a try?